helix-core/src/test/java/org/apache/helix/common/ZkTestBase.java

```diff
@@ -719,6 +722,20 @@ public void cleanupLiveInstanceOwners() {
       clientMap.clear();
     }
     _liveInstanceOwners.clear();
+
+    // wait 2 seconds for threads to be teared down.
+    // this reduce false alert.
+    Thread.sleep(1000);
```

Comment:
Are we still using thread.sleep? At least, we should have periodical check, for example check every 10 milisecond.

Reply:
This does not apply here. Tearing down thread is async to thread shutdown by the Java virtual machine. There is not way for use to check when the Java virtual machine really tear it down.

that said, I found by practice, waiting 2 sec does not really help much either. I will remove it a little bit later.

Reply:
removed.